roles/servicetelemetry/templates/manifest_smartgateway_events.j2

```diff
@@ -1,21 +1,35 @@
-apiVersion: smartgateway.infra.watch/v2alpha1
+apiVersion: smartgateway.infra.watch/v2
 kind: SmartGateway
 metadata:
   name: '{{ this_smartgateway }}'
   namespace: '{{ meta.namespace }}'
 spec:
+{% if this_collector.debug_enabled is defined and this_collector.debug_enabled %}
+  logLevel: "debug"
+{% else %}
+  logLevel: "info"
+{% endif %}
+  handleErrors: true
```

Comment:
I'm sure there is a single line version of this. I tried to get it to work but was running in to some syntax error. It makes the file a bit uglier but meh.